src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Popup/MessageDialog.xaml.cs

```diff
@@ -71,6 +71,12 @@ public MessageDialog()
 				Task.Run(async () => await messageDialog.ShowAsync());
 			};
 
+			WithEscapedCharacters.Tapped += (snd, evt) =>
+			{
+				var messageDialog = new Windows.UI.Popups.MessageDialog("\"Sample \\\"force escape test\\\" \\n \\t \\r continued sample.\"");
+				
+				Task.Run(async () => await messageDialog.ShowAsync());
```

Comment:
This is most likely to fail on iOS/Android because of a threading issue. Use `Dispatcher.RunAsync` instead.

Reply:
Actually I fixed this at some point, it works from a background thread now https://github.com/unoplatform/uno/commit/43a2189020e29aeff066c7b83e7ff808177820cf